test/test_max.js

```diff
@@ -0,0 +1,55 @@
+require('../dist/array_extensions');
+var assert = require('chai').assert;
+var sinon = require('sinon');
+var jsonfile  =require('jsonfile');
+
+var file = './fixtures/data.json';
+var fixtures = jsonfile.readFileSync(file);
+
+
+describe('Array #max', function () {
+	it('Should return empty', function () {
+		//Execute
+		var result = fixtures.arrayEmpty.max();
+
+		//Compare
+		assert.equal(result, null);
+	});
+
+	it('Should use comparator to find max element', function() {
+		//Setup
+		var expected = { "name": "pablo", "age": 20 };
+		var callback = function(a, b) {
```

Comment:
the most common and then accepted and recommended way is as follows:
```js
const callback = (a, b) => a.age - b.age;
```
Use `const` or `let`